roles/icingaweb2_modules/tasks/install_module.yml

```
@@ -70,6 +70,6 @@
       ansible.builtin.unarchive:
         src: "{{ _module.url }}"
         dest: "{{ icingaweb2_modules_path }}/{{ _module_name }}"
-        remote_src: true
+        remote_src: "{{ _module.remote_source | default('true') }}"
```

We don't want a string `'true'` here but a boolean `true` instead.

```suggestion
        remote_src: "{{ _module.remote_source | default(true) }}"
```